utils/e2e-tests/ts/tests/rpc/eth.ts

```diff
@@ -28,6 +28,23 @@ describe("eth rpc", () => {
     expect(network.chainId).toBe(5234n);
   });
 
+  it("has the expected balance value", async () => {
+    const [alice, _] = devSigners;
+    const charlie = ethers.ethers.Wallet.createRandom();
+    const transferBalance = 1000;
+
+    const tx = await alice.sendTransaction({
+      to: charlie.address,
+      value: transferBalance,
+    });
+
+    await tx.wait(1, 12000);
+
+    const charlieBalance = await provider.getBalance(charlie);
+
+    expect(Number(charlieBalance)).toBe(transferBalance);
```

Comment:
Consider using `bigint` (`1000n`) for balances and avoid conversion to regular numbers

Reply:
https://github.com/humanode-network/humanode/pull/764/commits/923ad76d0b25124977474621cb5f76b441b086c2